Jenkins_jobs/WorkspaceInfo.groovy

```diff
@@ -11,93 +12,97 @@ clones = [:]
  * 
  * The nodes can be either passed in or grabbed from Jenkins
  */
-stage('Pepare_Space_Monitoring_Jobs') {
-    NodeHelper nodeHelper = new NodeHelper();
-
-    String projectLabel
-    if (params.projectLabel.length() > 1) {
-        projectLabel = params.projectLabel
-    } else {
-        projectLabel = "all"
-    }
-    
-    ArrayList<Computer> computers = new ArrayList<Computer>()
-    if (params.machines.length() > 1) {
-        String[] inputMachines = params.machines.split(",")
-        for (String inputMachine : inputMachines) {
-            computers.add(nodeHelper.getComputer(inputMachine))
+    stage('Pepare_Space_Monitoring_Jobs') {
```

Comment:
Why was the indent added?

Reply:
ah, that was left over from when i had the stage enclosed in a node block. I'll remove it